### PR TITLE
Call modules Reload function on reload message from CLI

### PR DESCRIPTION
**Pull request by KristinaKoeva**

### CODE REVIEW

**Comment by KristinaKoeva** [comment]
run ci

**Comment by KristinaKoeva** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Comment by ivanbuhov** [comment]
Don't merge, yet. Review and comments are welcome.

File: src/NativeScript/inspector/InspectorPageAgent.mm [hunk 1]
```diff
@@ -38,7 +40,18 @@
 }
 
 void InspectorPageAgent::reload(ErrorString&, const bool* in_ignoreCache, const String* in_scriptToEvaluateOnLoad) {
-    ASSERT_NOT_REACHED();
+    JSValue liveSyncCallback = m_globalObject->get(m_globalObject.globalExec(), Identifier::fromString(&m_globalObject.vm(), "__onLiveSync"));
+    if (!liveSyncCallback.isUndefined()) {
```
Comment (fealebenpae) [inline src/NativeScript/inspector/InspectorPageAgent.mm]: You don't need this check - `getCallData` already checks that the value is a Cell.

File: src/NativeScript/inspector/InspectorPageAgent.mm [hunk 2]
```diff
@@ -38,7 +40,18 @@
...
+        CallData callData;
+        CallType callType = getCallData(liveSyncCallback, callData);
```
Comment (fealebenpae) [inline src/NativeScript/inspector/InspectorPageAgent.mm]: You should check whether `callType` is `CallTypeNone` and throw an exception through the inspector controller if so.

**Pull request merged**